src/main/java/ai/aitia/arrowhead/application/library/ArrowheadService.java

```diff
@@ -584,6 +594,54 @@ public boolean closeWSConnection(final String wsManagerId) {
 		return false;
 	}
 
+	//-------------------------------------------------------------------------------------------------
+	/**
+	 * Connect to MQTT broker.
+	 *
+	 * @param handler string id of the WebSocketcallback handler
+	 * @param brokerAddress address to the broker
+	 * @param port port to the broker
+	 * @param clientId the client name to use
```

Comment:
`password` is missing
`caFile`, `certFile`, `keyFile` is also missing but these coud be retrieved from `SSLProperties` component